ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -333,17 +323,14 @@ def test_source_ipahealthcheck_ipa_host_check_ipahostkeytab(self):
             "Minor code may provide more information, "
             "Minor (2529638972): Generic error (see e-text)"
         )
-        dirsrv_ipactl_status = 'Directory Service: STOPPED'
+        dirsrv_ipactl_status = "Directory Service: STOPPED"
         api.env.realm = self.master.domain.name
         serverid = (realm_to_serverid(api.env.realm)).upper()
         dirsrv_service = "dirsrv@%s.service" % serverid
         self.master.run_command(["systemctl", "stop", dirsrv_service])
-        result = self.master.run_command(
-            ["ipactl", "status"])
+        result = self.master.run_command(["ipactl", "status"])
```

Comment:
Are we using `result` anywhere else?

Reply:
@mrizwan93 removed the result since we are not using it.